src/parsers/YoutubeParser.ts

```diff
@@ -102,7 +102,7 @@ class YoutubeParser extends Parser {
                 id: video.id,
                 url: url,
                 title: video.snippet.title,
-                description: video.snippet.description,
+                description: this.escapeTagsInYouTubeDescription(video.snippet.description, this.settings.youtubeDescTagEscape),
```

Comment:
Ternary operator is more elegant because you don't have to pass additional argument to escapeTagsInYouTubeDescription method.

`this.settings.youtubeDescTagEscape ? this.escapeTagsInYouTubeDescription(video.snippet.description) : video.snippet.description`